clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -103,25 +98,6 @@ object ClioCommand extends ClioParsers {
   val moveGvcfName = "move-gvcf"
   val deleteGvcfName = "delete-gvcf"
 
-  /*
-   * Based on its docs / examples, caseapp *should* be able to inductively
-   * derive a `CommandParser[ClioCommand]` without needing this import. For
-   * some reason, though, the implicit values defined in the companion object
-   * of `CommandParser` aren't getting picked up during implicit search like
-   * they should be (on a clean compile in a new JVM at least; something about
-   * the Zinc incremental compiler used by the SBT repl and IntelliJ's Scala
-   * plugin makes compilation work the second time around, even though if you
-   * run "show implicit parameters" on `CommandParser[ClioCommand]` below in
-   * IntelliJ without this import it displays a "no implicit found" error message).
-   *
-   * By importing the implicits from the companion object directly, we get
-   * induction to actually work. The `implicitly` call both demonstrates that
-   * the base-case implicit is properly in-scope and convinces IntelliJ that
-   * the import is actually used for something.
-   */
-  import caseapp.CommandParser._
-  implicitly[CommandParser[CNil]]
```

Comment:
clean-compile from a cold SBT is working for me without this now :tada:

Somebody else might want to check that it works for them before we delete it.